v3.0-RC2/vehicle_status.json

```
@@ -0,0 +1,156 @@
+      "type": "integer",
+      "minimum": 0
+    },
+    "version": {
+      "description":
+        "GBFS version number to which the feed conforms, according to the versioning framework (added in v1.1).",
+      "type": "string",
+      "enum": [
+        "3.0-RC",
+        "3.0-RC2",
+        "3.0"
+      ]
+    },
+    "data": {
+      "description":
+        "Array that contains one object per vehicle as defined below.",
+      "type": "object",
+      "properties": {
+        "vehicles": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "vehicle_id": {
+                "description": "Rotating (as of v2.0) identifier of a vehicle.",
+                "type": "string"
+              },
+              "lat": {
+                "description": "The latitude of the vehicle.",
+                "type": "number",
+                "minimum": -90,
+                "maximum": 90
+              },
+              "lon": {
+                "description": "The longitude of the vehicle.",
+                "type": "number",
+                "minimum": -180,
+                "maximum": 180
+              },
+              "is_reserved": {
+                "description": "Is the vehicle currently reserved?",
+                "type": "boolean"
+              },
+              "is_disabled": {
+                "description": "Is the vehicle currently disabled (broken)?",
+                "type": "boolean"
+              },
+              "rental_uris": {
+                "description": "Contains rental uris for Android, iOS, and web in the android, ios, and web fields (added in v1.1).",
+                "type": "object",
+                "properties": {
+                  "android": {
+                    "description": "URI that can be passed to an Android app with an intent (added in v1.1).",
+                    "type": "string",
+                    "format": "uri"
+                  },
+                  "ios": {
+                    "description": "URI that can be used on iOS to launch the rental app for this vehicle (added in v1.1).",
+                    "type": "string",
+                    "format": "uri"
+                  },
+                  "web": {
+                    "description": "URL that can be used by a web browser to show more information about renting this vehicle (added in v1.1).",
+                    "type": "string",
+                    "format": "uri"
+                  }
+                }
+              },
+              "vehicle_type_id": {
+                "description": "The vehicle_type_id of this vehicle (added in v2.1-RC).",
+                "type": "string"
+              },
+              "last_reported": {
+                "description": "The last time this vehicle reported its status to the operator's backend in POSIX time (added in v2.1-RC).",
```

Let's please update the description to [RFC3339 format](https://www.rfc-editor.org/rfc/rfc3339).

Thank you!

```suggestion
                "description": "The last time this vehicle reported its status to the operator's backend in RFC3339 format (added in v2.1-RC).",
```